grapher/controls/entityPicker/EntityPicker.tsx

```diff
@@ -114,32 +119,39 @@ export class EntityPicker extends React.Component<{
         return this.manager.entityPickerColumnDefs ?? []
     }
 
-    @computed private get metricOptions() {
-        return this.pickerColumnDefs.map((col) => {
+    @computed private get metricOptions(): { label: string; value: string }[] {
+        return this.pickerColumnDefs.map((col): {
+            label: string
+            value: string
+        } => {
             return {
                 label: col.name || col.slug,
                 value: col.slug,
             }
         })
     }
 
-    @computed private get activePickerMetricColumn() {
+    @computed private get activePickerMetricColumn(): CoreColumn | undefined {
         if (!this.metric) return undefined
         return this.manager.entityPickerTable?.getColumns([this.metric])[0]
     }
 
-    @computed private get availableEntitiesForCurrentView() {
+    @computed private get availableEntitiesForCurrentView(): Set<string> {
         if (!this.manager.requiredColumnSlugs?.length || !this.grapherTable)
             return this.selection.availableEntityNameSet
         return this.grapherTable.entitiesWith(this.manager.requiredColumnSlugs)
     }
 
     @computed
-    private get entitiesWithMetricValue(): EntityOptionWithMetricValue[] {
+    private get entitiesWithMetricValue() {
```

Comment:
You _removed_ the return type here?

Reply:
Thanks for catching this! I tried to automate some of the annotations which occasionally removed an annotation and I missed this one.